### PR TITLE
doc example: header based latency tracking

### PR DESCRIPTION
**Pull request by oschaaf**
Another example, to make it easier to discover how to accomplish
tracking of latency communicated through response headers when
using Nighthawk.

Signed-off-by: Otto van der Schaaf <ovanders@redhat.com>

### CODE REVIEW

**Comment by oschaaf** [comment]
Please do not merge before I have amended the sign-off email to my redhat.com email address.

**Review by mum4k — APPROVED** [review]
Nice, this is very useful.

**Pull request merged**
